### DISCUSSION TITLE
Svelte tutorial

### DISCUSSION THREAD

**Discussion by gbuerk** [noam-honig/radweb#78] [Ideas]
I just discovered this awesome project after reading about it in a TLDR newsletter.  I'm using it now with Svelte as my frontend and I've had a really great experience so far.  Well done, team!

I noticed there is an Angular, Vue, and React tutorial, but not Svelte yet.  I'd love to write one.  There were only a few little gotchas that were different from the Vue tutorial which is basically what I used to get up and running.

I'll submit a PR when I have it.  Mostly, I just wanted to post to say thanks for this awesome library!

**Comment by yoni-rapoport** [comment]
Thanks a lot for the feedback. Really glad to hear this.

If you run into any issues - we're [here](https://discord.gg/GXHk7ZfuG5).

Looking forward to your PR (no pressure :wink:).

**Comment by noam-honig** [comment]
Quick update: there's a great sveltekit tutorial written By @jycouet
You can find it at:
https://remult.dev/tutorials/sveltekit/
